src/featureParameters/IsEnabled_CustomRDSchedules.featureParameterBoolean

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As described in the other note, this will be come `DataCount_ONETIME_Use.featureParameterInteger`. I think a label such as "Data Count: Short Term Use Telemetry" will work. It's not visible outside of the Packaging Org anyway.